src/cmd/ksh93/edit/vi.c

```diff
@@ -1060,8 +1060,7 @@ static int cntlmode(Vi_t *vp)
 		refresh(vp,CONTROL);
 		vp->repeat = 1;
 	}
-/* NOTREACHED */
-	return(0);
+	UNREACHABLE();
```

Comment:
Either the original `/* NOTREACHED */` comment was wrong, or lines 1060-1061 are dead code. Seems to be that can be reached if the `ed_getchar()` encounters EOF...